services/console-proxy/server/src/com/cloud/consoleproxy/ConsoleProxy.java

```diff
@@ -328,7 +329,7 @@ public static void start(Properties conf) {
         }
 
         if (httpListenPort != 0) {
-            startupHttpMain();
+            startServerForNoVNC();
```

Comment:
Can you keep the `startupHttpMain` function and move the logic of `startServerForNoVNC` into `startupHttpMain` since we are no longer just doing VNC, we are also doing ajax through the same jetty service